src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1521,6 +1521,48 @@ static void log_cb (const rd_kafka_t *rk, int level,
         CallState_resume(cs);
 }
 
+static void oauth_cb (rd_kafka_t *rk, const char *oauthbearer_config, void *opaque) {
+	Handle *h = opaque;
+	PyObject *eo = NULL, *result = NULL;
```

Comment:
No need for initialization of eo, result, cs, token and expirty, err_code, they're always set below before being used.